tests/x11/tomcat.pm

```diff
@@ -63,8 +64,6 @@ sub run() {
     my $websocket_test = Tomcat::WebSocketsTest->new();
     $websocket_test->test_all_examples();
 
-    assert_screen('generic-desktop');
```

Comment:
This needle was here and there was no problem until the bad `generic-desktop` needle was created. I guess it can stay

Reply:
Keep it as it is, as we discussed.